src/parsers/xml-processor/class-xmlunsupportedexception.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * XML API: XMLUnsupportedException class
+ */
+
+namespace WordPress\XML;
+
+use Exception;
+
+/**
+ * Core class used by the XML processor during XML parsing
+ * for indicating that a given operation is unsupported.
+ *
+ * This class is designed for internal use by the XML processor.
+ *
+ * The XML API aims to operate in compliance with the XML5
```

Comment:
Did you mean XML 1.1?

Reply:
Ah, I meant XML 1.0. Not sure where 5 came from